### DISCUSSION TITLE
Indicators for the lffi publication ⚡

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#363] [Ideas] [answered]
I thought all the indicators you put in the program were really cool.

Example:
![Screenshot_2](https://user-images.githubusercontent.com/76891864/169540721-a858375a-8648-4220-9b92-a4ae07bdf73d.png)

Would it be possible to also put it in the lffi publication?

**Comment by sircharlo** [comment, marked as answer]
Paragraph numbers can be extracted automatically for Watchtower articles because all paragraphs are numbered. The _lff_ book does not have numbered paragraphs per se, so I'm afraid it isn't feasible, at least for the moment.
